roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -12,5 +12,12 @@
 - name: start aci-gw container
   service: name=aci-gw state=started
 
+- name: Contiv | Wait for netmaster
+  action: shell netctl tenant ls
+  register: tenant_result
+  until: tenant_result.stdout.find("default") != -1
+  retries: 5
+  delay: 5
```

Comment:
I have seen that 25 sec is not enough in some cases. Please use 9 retries with a 10 sec delay or 18 retries with a 5 sec delay. Note that it will break out of the loop as soon as it succeeds. So, there is no impact when it's successful.